src/components/icon/icon.spec.js

```diff
@@ -251,14 +274,6 @@ describe("Icon component", () => {
             },
             wrapper.toJSON()
           );
-
```

Comment:
Please can you add positive assertions for the different variants by making the wrapper interactive instead of deleting the test.

Reply:
I'm not able to call `assertStyleMatch` if there are no styles.
Will it make sense?
<img width="632" alt="Zrzut ekranu 2021-05-28 o 13 11 49" src="https://user-images.githubusercontent.com/77274590/119975554-4b1bd380-bfb6-11eb-9abd-808f28ffbf04.png">

Reply:
> making the wrapper interactive
If we add a tooltip, these tests should still pass... I think.

Reply:
ahhh that was a tricky one :D changed, thanks!